open_bounties/0000 Xrpl.js Open Bounties.md

```diff
@@ -0,0 +1,62 @@
+---
+ID: "0000"
+Discussion: https://github.com/XRPLBounties/Proposals/discussions/0000
+Category: Core library support
+Status: Open
+Amount: Variable
+---
+
+# Xrpl.js Open Bounties
+
+**Category: Open Source Tooling**
+
+<!--
+Recommend a total value amount for the bounty, in U.S. Dollars. The exact amount will be determined by the approving committee and may be higher or lower than this recommendation.
+-->
+
+**Amount:** $250 - $1000+ per bounty
+
+## Overview
+
+<!--
+Please provide the context required to complete the bounty.
+Questions you should answer here:
+1. What is the high level explanation of this bounty? (1-3 sentences)
+2. What problem is this solving?
+3. What are the requirements for this solution?
+-->
+
+If an open issue in xrpl.js has the tag "Approved Bounty", it is eligible to be completed for the specified dollar amount.
+These tasks generally take at least a day of coding, are reasonable to build on your own, and are a way to learn about
+a specific piece of the XRPL ecosystem.
+
+In order to complete the bounty, your PR must include unit tests, be well documented, and be merged in to the xrpl.js repository.
```

Comment:
edit changlog.md

Reply:
> I think in addition to labeling issues as "Open Bounties", we still need to list them here, each with requirements (if clarification is needed in addition to the issue) and award amount.

Agreed. We'll need a couple examples to do that though, so will likely have to wait until we've finished grooming past issues for good bounties.